mscore/plugin/pluginCreator.cpp

```diff
@@ -65,6 +65,9 @@ PluginCreator::PluginCreator(QWidget* parent)
       actionSaveAs->setIcon(*icons[int(Icons::fileSaveAs_ICON)]);
       fileTools->addAction(actionSave);
 
+      actionRun->setShortcut(Qt::Key_F5);
+      actionStop->setShortcut(Qt::Key_F7);
+
```

Comment:
This is the only change vs. the prev PR, which already got merged, can you rebase?

I believe somtehing like
`git pull --rebase Jojo-Schmitz/MuseScore 3.x`

Reply:
Those shortcuts don't work (again)

Reply:
Also I wonder whether they'd better include Ctrl

Reply:
Sorry, I've been too slow with my comments ;-)

Reply:
Weird. They seem to work for me. I'm not going to mess with it because I have a build I'm working on for something else, but I'll come back to it later. Got a rebase though.

Reply:
I get this `Warning: QAction::event: Ambiguous shortcut overload: F5 (:0, )`

Reply:
Oh it's because now some how the removal was lost: 
These need to be removed, and some how that was lost between merge and this thing. Will do a PR a little after I do some other things here

```cpp
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
```

Reply:
These lines need to be removed, some how that was lost between last merge and this commit..: 

```javascript

         <property name="shortcut">
          <string>F5</string>
         </property>
        </widget>
       </item>
       <item>
@@ -61,6 +64,9 @@
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="shortcut">
          <string>F7</string>
         </property>
```

Reply:
With those removed it doesn't even compile

Reply:
Maybe not, I managaed to confuse myself

Reply:
Good to go?

Reply:
Yep, manually merged it

Reply:
Ok, cool.